emcc.py

```diff
@@ -395,6 +395,23 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  apply_min_browser_versions(user_settings)
```

Comment:
Can you move this to the end of `phase_linker_setup`?   The reason is that browser settings only apply to the link phase and are not relevant during compilation.